### ISSUE TITLE
Allow Rule objects to be used

### ISSUE THREAD

**Issue by ash-jc-allen** [ash-jc-allen/laravel-config-validator#11]
Add the moment, rules can only be passed in as a string (example - ``` required ```). We will also need the functionality to handle Rule objects.

https://laravel.com/docs/8.x/validation#using-rule-objects

**Comment by mbroadhead** [comment]
Did this get fixed? I'm still seeing the issue on v1.3.0. It appears the fix is as easy as assigning the rules directly without imploding them:
```patch
diff --git a/src/Services/ValidationRepository.php b/src/Services/ValidationRepository.php
index c2af243..391202b 100644
--- a/src/Services/ValidationRepository.php
+++ b/src/Services/ValidationRepository.php
@@ -49,7 +49,7 @@ class ValidationRepository
             $configKey = $key.'.'.$rule->getFieldName();

             // Add the rules for the field to the repository.
-            $this->rules[$configKey] = implode('|', $rule->getRules());
+            $this->rules[$configKey] = $rule->getRules();

             // Add the current config values for the field to the repository.
             $this->fetchCurrentConfigValues($key, $rule);
```

**Comment by ash-jc-allen** [comment]
Hey @mbroadhead! I've not really touched this package in a while since I first wrote it. So, I can't quite remember if this was added properly.

I'll try and give this a check over the next few days if it's something that seems to be broken. I'm also hoping to update this package in the near future because, at the moment, it seems to break the `artisan config:cache` command.

Thanks for the suggestion though, you've probably saved me quite a bit of debugging 😄
